lib/libutee/tee_user_mem.c

```diff
@@ -256,7 +257,7 @@ static int check_elem(struct user_mem_elem *ap)
 	return check_elem_end(ap);
 }
 
-/* Trap PC element are corrupted. */
+/* In debug mode, trap PC element are corrupted. */
```

Comment:
should better keep comment unchanged and replace `assert()` below with `TEE_ASSERT()`